service/resources/helpers/helpers.py

```diff
@@ -50,6 +50,11 @@ def get_emails(emails, email_type):
         email_list = []
         counter = 0
         for email in emails:
+            if email_type != 'to' and \
+               'email' not in email or 'name' not in email:
+                print(f'Missing "email" or "name" for email of type "{email_type}" - skipping: {email}')
+                continue
+
             if email_type == 'to':
                 email_list.append(To(email['email'], email['name'], p=counter))
```

Comment:
The direct access to `email['name']` could be problematic in the future as according to the docs, it should be optional. I was going to do something like a `.get('name', None)` (for this and the CC/BCC), but didn't wanna bloat this PR.. so if we agree, I could make a ticket to make it more resilient.

Reply:
See previous comment about the `name` field. Maybe we can fill it with `email` if it is not there.

Reply:
Addressed in [commit](https://github.com/SFDigitalServices/email-microservice-py/pull/21/commits/42194d533ca357a47ae73b29965cbbcb7513a365).